### ISSUE TITLE
Unable to deploy email template; 'verify_email_by_code'

### ISSUE THREAD

**Issue by Jelle-vz** [auth0/auth0-deploy-cli#303]
### Description

When adding the email template for 'verify_email_by_code' to the deployment directory, the pipleline fails, saying:

```bash
2021-01-06T13:04:19.197Z - debug: Error: Schema validation failed loading [
    {
        "keyword": "enum",
        "dataPath": ".emailTemplates[6].template",
        "schemaPath": "#/properties/emailTemplates/items/properties/template/enum",
        "params": {
            "allowedValues": [
                "verify_email",
                "reset_email",
                "welcome_email",
                "blocked_account",
                "stolen_credentials",
                "enrollment_email",
                "change_password",
                "password_reset",
                "mfa_oob_code"
            ]
        },
        "message": "should be equal to one of the allowed values"
    }
]
```

### Reproduction

* Add email template `verify_email_by_code.html` & `verify_email_by_code.json` to emails directory
* deploy to a tenant using the `auth0-deploy-cli` (using v6.0.0)
* Check debug log.

### Environment

- **"auth0-deploy-cli": "^6.0.0"**
- **Gitlab deployment pipeline**

**Comment by pmalouin** [comment]
Thanks for reporting @Jelle-vz and sorry for the experience. I'm working on addressing this.

In the mean time, you can skip that template on import by manually removing it from the YAML file or the /emails directory.

**Comment by Jelle-vz** [comment]
HI @pmalouin, 
Thats great! Sure, we copy/paste the template manually now so we are not (really) blocked haha.

Do you have any indication on when we can expect the cli-tool to be updated?

Cheers!

**Comment by pmalouin** [comment]
> Do you have any indication on when we can expect the cli-tool to be updated?

Things are in motion, but we have some quality review cycles to go through before we publish it. It can go fast if I can get some eyes on this, but could take a few days if people are busy.

**Comment by pmalouin** [comment]
@Jelle-vz Fix is available starting with `auth0-deploy-cli@5.5.0+` ✔️ thanks for your patience

**Comment by Jelle-vz** [comment]
Thanks!
